### DISCUSSION TITLE
minecraft refuses to load...

### DISCUSSION THREAD

**Discussion by Donastria** [FabricMC/fabric-api#2324] [Mod Dev Support]
i finally made the switch! but I cannot seem to get Minecraft to load, which is why today was the day I sifted through EVERYTHING to make it. it wasn't loading, vut with a different error. Can you please help me get back to Minecrat???
[crash-2022-06-14_20.23.03-client.txt](https://github.com/FabricMC/fabric/files/8904685/crash-2022-06-14_20.23.03-client.txt)
.

**Comment by haykam821** [comment]
This crash seems to be caused by [Chassis](https://www.curseforge.com/minecraft/mc-mods/chassis), which is required by [Compressor](https://www.curseforge.com/minecraft/mc-mods/compressor). I would recommend opening an issue on [the Chassis issue tracker](https://github.com/EvergoodTeam/Chassis/issues).

**Comment by Donastria** [comment]
I would love to give you kudos and say you fixed it, but NOPE. Still not loading and I removed both mods so it wasn't that to begin with.... or it would be working but isn't..
[latest.log](https://github.com/FabricMC/fabric/files/8905216/latest.log)
.

**Reply by warjort** [reply]
````
[22:31:09] [Render thread/ERROR]: Unhandled game exception
java.lang.IllegalStateException: Config errors were found:

java.lang.IllegalArgumentException: Jankson file reading for "C:\Users\donas\AppData\Roaming\.minecraft\config\byg\trades.json5" failed due to the following error(s):
java.lang.IllegalArgumentException: Jankson file reading for "C:\Users\donas\AppData\Roaming\.minecraft\config\byg\trades.json5" failed due to the following error(s):
"sewingkit:tailor" is not a valid id in registry: Registry[ResourceKey[minecraft:root / minecraft:villager_profession] (Experimental)].
````

Something wrong with your "biomes you will go" trade config?
My guess is it is because you don't actually have a mod called "sewingkit" and therefore no tailor villager.

If you don't know how to modify the json5 file, you can try deleting that file (copy it for backup first) and it will probably recreate it from the mods you do have installed. Otherwise you will have to ask for help on the byg discord.

**Comment by Donastria** [comment]
I DO have sowing kit... I remember they said they snagged the logo form like Marco Vech or somrthing... that can't be it... here is the file. I am nt quite understanding... yesterday you mentioned a mod I KNEW I had deleted because it came up in the fabric launcher. I double checked and it WAS listed, but wasn't in my mods folder, because I had deleted it, and KNEW I had. But good anyway, I found what it was attached to and deleted it too. Now it is NOT lising mods I KNOW I have... help!!!???
[Sewing Kit From My Mods Folder.zip](https://github.com/FabricMC/fabric/files/8909041/Sewing.Kit.From.My.Mods.Folder.zip)
e

**Comment by Donastria** [comment]
And maybe the issue is just byg in general... I just tried to load again and byg threw a different error, but magically it lists sewing kit again like the ten logs before the last one where it magically was missing.. Here is the new log..
[latest.log](https://github.com/FabricMC/fabric/files/8909221/latest.log)
.

**Reply by warjort** [reply]
Your latest log shows the exact same error as before.
Like I said before. You should ask on byg discord for help if you don't know how to fix the json file.

From what I can tell  sewingkit is a forge mod. I don't see any fabric download on curseforge.
https://www.curseforge.com/minecraft/mc-mods/sewing-kit/files
